# Запуск цієї программи призведе до її паніки, виводу
# повідомлення помилки та сліду горутина, та визоду з
# ненульовим статусом.
$ go run panic.go
panic: маємо проблему

goroutine 1 [running]:
main.main()
	/.../panic.go:12 +0x47
...
exit status 2

# Зауважте що навідміну від інших мов що викоирстовують
# виключення для опрацювання помилок, в GO ідіоматично викорисовувати
# повернення помилок що вказують на проблеми як умога частіше.
